ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -570,6 +570,42 @@ def Conv3D(
     )
 
 
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    use_cudnn_on_gpu=False,
+    explicit_paddings=[],
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D"
+
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 3 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+
+    return tf_frontend.nn.conv2d(
+        input,
+        filter,
+        strides,
+        padding,
+        explicit_paddings,
```

Comment:
is this the correct definition of padding function, i will commit only after your approval
 
```
def explicit_padding(input, paddings):
    # paddings: (top, bottom, left, right) or (top, bottom, left, right, front, back)
    input_shape = ivy.shape(input)
    if len(input_shape) == 4:
        # 2D convolution
        paddings = [(0, 0), (0, 0), (paddings[0], paddings[1]), (paddings[2], paddings[3])]
    elif len(input_shape) == 5:
        # 3D convolution
        paddings = [(0, 0), (0, 0), (paddings[0], paddings[1]), (paddings[2], paddings[3]), (paddings[4], paddings[5])]
    else:
        raise ValueError('Invalid input shape: {}'.format(input_shape))
    return ivy.pad(input, paddings, mode='constant')
```

Reply:
or this implementation, this will be able to handle the NONE value for explicit_padding as well

````
def explicit_padding(
    padding: Union[str, Tuple[int, int], List[Union[int, Tuple[int, int]]]],
    kernel_size: Tuple[int, int],
) -> Optional[List[Tuple[int, int]]]:
    if padding == "SAME":
        return [(k // 2, k // 2 + k % 2) for k in kernel_size]
    elif padding == "VALID":
        return [(0, 0)] * 2
    elif padding == "EXPLICIT":
        return None
    elif isinstance(padding, tuple):
        if len(padding) == 2:
            padding = [padding] * 2
        return padding
    elif isinstance(padding, list):
        if not padding:
            return None
        if len(padding) == 2:
            padding = [padding] * 2
        return padding
    else:
        raise ValueError(f"Unsupported padding value: {padding}")
```